Jenkinsfile

```diff
@@ -551,42 +567,36 @@ parallel rocm20_ubuntu:
 
 rocm20_ubuntu_gfx906:
 {
-    try
-    {
-        node( 'docker && rocm20 && gfx906')
-        {
-        def hcc_docker_args = new docker_data(
-            from_image:'rocm/dev-ubuntu-16.04:2.0',
-            build_docker_file:'dockerfile-build-ubuntu-rock',
-            install_docker_file:'dockerfile-install-ubuntu',
-            docker_run_args:'--device=/dev/kfd --device=/dev/dri --group-add=video',
-            docker_build_args:' --pull' )
-
-        def hcc_compiler_args = new compiler_data(
-            compiler_name:'hcc-rocm19-ubuntu',
-            build_config:'Release',
-            compiler_path:'/opt/rocm/bin/hcc' )
-
-        def rocblas_paths = new project_paths(
-            project_name:'rocblas-ubuntu',
-            src_prefix:'src',
-            build_prefix:'src',
-            build_command: './install.sh -c' )
-
-        def print_version_closure = {
-          sh  """
-              set -x
-              /opt/rocm/bin/hcc --version
-            """
-          }
+  node( 'docker && rocm20 && gfx906' )
+  {
+    def hcc_docker_args = new docker_data(
+        from_image:'rocm/dev-ubuntu-16.04:2.0',
+        build_docker_file:'dockerfile-build-ubuntu-rock',
+        install_docker_file:'', //'dockerfile-install-ubuntu', Don't build installer
+        docker_run_args:'--device=/dev/kfd --device=/dev/dri --group-add=video',
+        docker_build_args:' --pull' )
 
-        build_pipeline( hcc_compiler_args, hcc_docker_args, rocblas_paths, print_version_closure )
-        }
-    }
-    catch( err )
-    {
```

Comment:
Why did you remove  the try catch? Andrews seemed to want it to distinguish gfx900/906. I didn't think it was the best way to do it, but I added until I can clean things up.